### PR TITLE
Feature/main structure

### PR DESCRIPTION
**Pull request by sp3c1**

### CODE REVIEW

File: config/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                        "httpProxy": "173.208.103.109:8800",
+                        "sslProxy": "173.208.103.109:8800",
+                    }
+                }
+            }
+        ]
+    ]
+};
+
+module.exports = conf;
```
Comment (mtjburton) [inline config/index.js:39]: No TMS integration 😉

File: index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+var server = require('https').Server(credentials, app);
+var io = require('socket.io')(server);
+var proxyCaps = require('./splitter');
+var config = require('./config');
+var bodyParser = require('body-parser');
+config.seleniumClients = proxyCaps;
+
+var _ = require('lodash');
+
+var mng = require('./mng')(config);
```
Comment (mtjburton) [inline index.js:17]: And you have a problem with cb and opts?

File: config/index.js (hunk 1 above)
Comment (sp3c1) [inline config/index.js:39]: yup

File: index.js (hunk 2 above)
Comment (sp3c1) [inline index.js:17]: lol, when i m prototyping on my knee i dont care. When I have time I try to avoid ;]

**Pull request merged**